CHANGELOG.md

```diff
@@ -8,10 +8,11 @@ All notable changes to the ZSS package will be documented in this file.
 
 - Enhancement: Add an endpoint for PassTicket generation
 - Enhancement: Add an endpoint for user info
+- Enhancement: Added method to read and set loglevel of dataservices
 - Bugfix: Unixfile Copy and Rename API doesn't support distinct error status response
 - Bugfix: Dataset contents API returns invalid error status for undefined length dataset update request
-- Bugfix: Unixfile API returns valid error status
```

Comment:
I'd like to know why did you remove the line?

Reply:
It is duplicate of "- Bugfix: Unixfile Copy and Rename API doesn't support distinct error status response".